arch/arm/mach-aspeed/ast2700/cpu-info.c

```diff
@@ -47,9 +46,9 @@ void ast2700_print_soc_id(void)
 #define SYS_SRST		BIT(0)
 
 #define WDT_RST_BIT_MASK(s)	(GENMASK(3, 0) << (s))
-#define BIT_WDT_SOC(s)		(BIT(0) << (s))
+#define BIT_WDT_SOC(s)		(BIT(2) << (s))
```

Comment:
how are these changes related to POR?

Reply:
Part of aspeed repo changes, cherrypicked origin commit in same pull request